ui/components-lib/src/styles/main.scss

```
@@ -1,10 +1,21 @@
 @use './_colors.scss' as *;
 // @import url('https://fonts.googleapis.com/css?family=Heebo:400,500,600,700&display=swap');
 
+html,
+body,
+#root {
+  height: 100%;
+}
+
 body {
   background: $background-light;
   font-family: 'Heebo', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Heebo" (font-family-name-quotes)

(font-family-name-quotes)